public/backend/js/register.js

```diff
@@ -0,0 +1,23 @@
+$('#register').on('click',function(e){
+	e.preventDefault();
+	$.ajaxSetup({
+        headers: {
+            'X-CSRF-TOKEN': $('meta[name="_token"]').attr('content')
+        }
+	});
+	$.ajax({
+	        type: 'POST',
+	        url: path_register,
+	        data: {email: $('form #emailregister').val(), pass: $('form #passregister').val()},
+	        dataType: "json",
+	        success: function (data) {			            
+	            if(data.allow=='true'){
```

Comment:
[rank A] in this case, if you cannot use boolean type, you can use number for instead. For example: 0 or 1 to allow or not allow.